narwhals/_pandas_like/series.py

```diff
@@ -944,3 +946,29 @@ def convert_time_zone(self, time_zone: str) -> PandasLikeSeries:
         else:
             result = self._pandas_series._native_series.dt.tz_convert(time_zone)
         return self._pandas_series._from_native_series(result)
+
+    def timestamp(self, time_unit: Literal["ns", "us", "ms"] = "us") -> PandasLikeSeries:
+        s = self._pandas_series._native_series
+        dtype = self._pandas_series.dtype
+        is_pyarrow_dtype = "pyarrow" in str(self._pandas_series._native_series.dtype)
+        mask_na = s.isna()
+        if dtype == self._pandas_series._dtypes.Date:
+            s_cast = s.astype("Int32[pyarrow]")
```

Comment:
That's because date can only be pyarrow backed, is that right?

Reply:
yup! will add a comment